### DISCUSSION TITLE
Multiple Time controls?

### DISCUSSION THREAD

**Discussion by Moonwalker316** [SebLague/Chess-Challenge#244] [Ideas]
Maybe there will be multiple time controls like 15 min, 10 min, 5 min, 2 min, 1 min and 10 second? This will encourage more intresting bots rather than the same min-max alpha-beta pruning bot.

**Comment by mcthouacbb** [comment]
How does changing the time-control encourage bots that don't use minimax? If anything, longer time-controls would only encourage minimax. Minimax is the probably the most well-known algorithm that can take advantage of extra thinking time, most other heuristics are relatively constant in time, so they wouldn't play any better with longer time controls, while minimax based bots will be able to think for longer.

**Comment by Moonwalker316** [comment]
Fair point, but tbh it would be intresting to see which bots break down with lower time controls. Bots that don't care about the time given will certainly run out of time in the 10 second matchup
